cypress/integration/spec.ts

```diff
@@ -38,10 +39,6 @@ context('ix-video', () => {
     cy.get(player).should('exist');
   });
 
-  it('should await for video.js instance to mount player', () => {
-    cy.get(player).should('have.attr', 'data-setup');
-  });
-
```

Comment:
Sure thing.

So instead of ending with markup that looks a little like this:

```html
<ix-video data-setup=`{"sources": [{...}]}`>
	<video data-setup=`{"sources": [{...}]}` />
</ix-video>
```

It looks like this:

```html
<ix-video data-setup=`{"sources": [{...}]}`>
	<video />
</ix-video>
```

This is a very simplified example, but what we're trying to avoid is setting `data-setup` on `ix-video` and `video`. Instead, we set it on `ix-video` and praise the string into JSON, using it to create the `options` object.